### DISCUSSION TITLE
build firmware failed

### DISCUSSION THREAD

**Discussion by dovantuan02** [LuckfoxTECH/luckfox-pico#142] [Q&A]
i am using rv1103 pico board. I am write app in app rkipc old. But when i was build all, console log appears as follows:
![image](https://github.com/user-attachments/assets/d7e1f8f1-6643-415e-9ac0-0cc4a3aec276)
Please help me !

**Comment by luckfox-eng33** [comment]
Did you modify the partition table or add too much code? At present, it seems that it is caused by the partition being too small.
